client/extensions/woocommerce/app/settings/email/email-settings/components/notifications-origin.js

```diff
@@ -12,36 +14,62 @@ import ListItemField from 'woocommerce/components/list/list-item-field';
 import FormTextInput from 'components/forms/form-text-input';
 import FormLabel from 'components/forms/form-label';
 import FormSettingExplanation from 'components/forms/form-setting-explanation';
-
-const NotificationsOrigin = ( { item, recipient, onChange, isPlaceholder } ) => (
-	<ListItem>
-		<ListItemField className="components__notification-origin">
-			{ ! isPlaceholder
-				? <FormLabel>
-					{ item.title }
-				</FormLabel>
-				: <p className="components__is-placeholder" />
-			}
-			<FormTextInput
-				className={ isPlaceholder ? 'components__is-placeholder' : null }
-				name={ item.field }
-				onChange={ onChange }
-				value={ recipient }
-			/>
-			{ ! isPlaceholder
-				? <FormSettingExplanation>
-					{ item.subtitle }
-				</FormSettingExplanation>
-				: <p className="components__is-placeholder" />
+import FormTextValidation from 'components/forms/form-input-validation';
+const NotificationsOrigin = ( { item, recipient, onChange, isPlaceholder, checkEmail, translate, placeholder } ) => {
+	const change = ( { target: { value } } ) => {
+		onChange(
+			{
+				setting: item.field,
+				option: item.option,
+				value,
 			}
-		</ListItemField>
-	</ListItem>
-);
+		);
+	};
+
+	const emailValidationError = checkEmail &&
+		( recipient.length !== 0 ) &&
+		! emailValidator.validate( recipient );
+
+	return (
+		<ListItem>
+			<ListItemField className="components__notification-origin">
+				{ ! isPlaceholder
+					? <FormLabel>
+						{ item.title }
+					</FormLabel>
+					: <p className="components__is-placeholder" />
+				}
+				<FormTextInput
+					className={ isPlaceholder ? 'components__is-placeholder' : null }
+					isError={ emailValidationError }
+					name={ item.field }
+					onChange={ change }
+					value={ recipient }
+					placeholder={ placeholder }
+				/>
+				{ emailValidationError && (
+					<FormTextValidation
+						isError={ true }
+						text={ translate( '%(recipient)s is not a valid email address.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( '%(email)s is not a valid email address.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).